client/src/App.tsx

```diff
@@ -1,5 +1,22 @@
+// import { RouterProvider } from 'react-router-dom';
```

Comment:
should we pass the router here :thinking:  ? ` const routing = useRoutes(ThemeRoutes);`

Reply:
Actually I am not sure how this will be since we have a fixed layout (sidebar, header) for all routes and a dynamic pages.

Here is an example of your idea but it changes the whole page not a part of it.
https://reactrouter.com/en/main/hooks/use-routes

I tried different solutions but nothing worked for me, could you help me out with this?